packages/flutter/test/widgets/selectable_region_test.dart

```diff
@@ -3594,6 +3594,20 @@ void main() {
     skip: kIsWeb, // [intended] Web uses its native context menu.
   );
 
+  testWidgets('Ensure consistent layout of SelectionArea children between platforms', (WidgetTester tester) async {
```

Comment:
This test passes on the latest `master`, so I think it should probably be more involved. I think making tests similar to the code samples in the related issues is a good place to start. Mainly the test should verify that the previously broken behavior is no longer broken. I think for this issue https://github.com/flutter/flutter/issues/148934 it is sufficient to verify that the `Text` widgets all begin at the same x axis `Offset`.